definitions/variable/emissions/emissions.yaml

```diff
@@ -43,28 +46,33 @@
     description: Emissions of {Level-3 Species} from soil management practices
       in the agriculture sector
     unit: "{Level-3 Species}"
+- Emissions|*|AFOLU|Biomass Burning:
+    description: Agricultural Waste Burning (DESCRIPTION TO BE UPDATED)
+    unit: "{Level-3 Species}"
 - Emissions|{Level-3 Species}|AFOLU|Land:
     description: Emissions of {Level-3 Species} from forestry and other land use
     unit: "{Level-3 Species}"
-- Emissions|{Level-3 Species}|AFOLU|Land|Wetlands:
-    description: Emissions of {Level-3 Species} from managed peatlands (drained and rewetted)
+- Emissions|{Level-3 Species}|AFOLU|Land|Peat Burning:
+    description: Emissions of {Level-3 Species} from managed peatlands (drained and rewetted)  (DESCRIPTION TO BE UPDATED)
     unit: "{Level-3 Species}"
 - Emissions|{Level-3 Species}|AFOLU|Land|Forest Burning:
-    description: Emissions and removals of {Level-3 Species} from land with woody vegetation
-      (forests, IPCC category 3B1)
+    description: Emissions and removals of {Level-3 Species} from land with woody vegetation 
+      (forests, IPCC category 3B1)  (DESCRIPTION TO BE UPDATED)
     unit: "{Level-3 Species}"
 - Emissions|{Level-3 Species}|AFOLU|Land|Grassland Burning:
     description: Emissions and removals of {Level-3 Species} from rangelands and pasture land
-      (IPCC category 3B3)
-    unit: "{Level-3 Species}"
-- Emissions|{Level-3 Species}|AFOLU|Land|Other Land Burning:
-    description: Emissions and removals of {Level-3 Species} from any other categories
-      of land burning (e.g. peatlands)
+      (IPCC category 3B3)  (DESCRIPTION TO BE UPDATED)
     unit: "{Level-3 Species}"
 
+- Gross Emissions|CO2|AFOLU:
```

Comment:
Thank you for adding this, but can you please move this to just below `Emissions|{Level-2 Species}|AFOLU` and then have an empty line for consistency of the ordering?

Reply:
418620d2ee025365922b15204cc4fb9d978869db